src/sys/signal.rs

```diff
@@ -1046,6 +1052,8 @@ mod sigevent {
                 SigevNotify::SigevThreadId{..} => libc::SIGEV_THREAD_ID,
                 #[cfg(all(target_os = "linux", target_env = "gnu", not(target_arch = "mips")))]
                 SigevNotify::SigevThreadId{..} => libc::SIGEV_THREAD_ID,
+                #[cfg(all(target_os = "linux", target_env = "uclibc"))]
```

Comment:
All of the references to `SIGEV_THREAD_ID` on Linux can probably be consolidated, but it's outside the scope of this PR. I see `SIGEV_THREAD_ID` defined by musl, but not in rust-lang/libc's redefinitions, so it'd need to be updated there first.

Not necessary to do it here, just calling it out.

Reply:
:+1: